neural_speed/models/llama/llama.cpp

```diff
@@ -399,9 +399,9 @@ static bool llama_model_eval_internal(model_context* ctx, const model_input* inp
   ne_build_forward_expand(&gf, inpL);
   ne_graph_compute(ctx0, &gf);
 
-  if (ns_log_level() == 0 || ns_log_level() == 2) {
-    ne_graph_profiling(&gf);
-  }
+  // if (ns_log_level() == 0 || ns_log_level() == 2) {
+  // ne_graph_profiling(&gf);
+  // }
```

Comment:
Why no profiling for llama?

Reply:
https://github.com/intel/neural-speed/pull/81

Reply:
fixed in this pr, merged